udata/core/topic/models.py

```diff
@@ -10,7 +8,7 @@
 __all__ = ('Topic', )
 
 
-class Topic(db.Document, db.Owned):
+class Topic(db.Document, db.Owned, db.Datetimed):
```

Comment:
I think you need to explicitly manage the `last_modified` property on api calls right?

Reply:
Yup https://github.com/opendatateam/udata/pull/2987/files#diff-66c97efd0f9b3994aeeb90485bafc1d004099de07ddfec978388731f455d38fc

Reply:
I mean to update the `last_modified` value on a PUT

Reply:
Oh, sorry. Isn't it handled by this signal? https://github.com/opendatateam/udata/blob/9142ed5ae2f4fa8258892da8d0b9ed70dba1997c/udata/models/datetime_fields.py#L66

Reply:
Indeed my bad! :ok_hand:
On topics that haven't been saved since this PR, last_modified will show as utcnow, but that's life!